changelogs/current.yaml

```diff
@@ -10,39 +10,40 @@ behavior_changes:
 - area: prefix_match_map
   change: |
     :ref:`prefix_match_map <envoy_v3_api_field_config.common.matcher.v3.Matcher.MatcherTree.prefix_match_map>`
-    now continues to search for a match with shorter prefix if a longer match
+    now continues to search for a match with a shorter prefix if a longer match
     does not find an action. This brings it in line with the behavior of ``matcher_list``.
     This change can temporarily be reverted by setting the runtime guard
     ``envoy.reloadable_features.prefix_map_matcher_resume_after_subtree_miss`` to ``false``.
     If the old behavior is desired more permanently, this can be achieved in config by setting
-    an ``on_no_match`` action that responds with 404 for each subtree.
+    an ``on_no_match`` action that responds with ``404`` for each subtree.
 - area: server
   change: |
     Envoy will automatically raise the soft limit on the file descriptors to the hard limit. This behavior
     can be reverted using the runtime guard ``envoy_restart_features_raise_file_limits``.
 - area: build
   change: |
-    Removed the clang-libstdc++ toolchain setup as this is no longer used or tested by the project.
-    Consolidated clang and gcc toolchains which can be used with ``--config=clang`` or ``--config=gcc``.
-    These use libc++ and libstdc++ respectively.
+    Removed the ``clang-libstdc++`` toolchain setup as this is no longer used or tested by the project.
+    Consolidated ``clang`` and ``gcc`` toolchains which can be used with ``--config=clang`` or ``--config=gcc``.
```

Comment:
i think these are names and/or acronyms rather than  literals - probably we should capitalize them correctly